.gitignore

```diff
@@ -12,3 +12,5 @@ system-test/*key.json
 google-cloud-logging-winston-*.tgz
 google-cloud-logging-bunyan-*.tgz
 package-lock.json
+.vscode
+yarn.lock
```

Comment:
Aren't we deleting lock files?

Reply:
yup!  This is just adding them to the .gitignore.